pkg/raft/raftpb/raft.go

```diff
@@ -22,3 +24,36 @@ type Epoch int64
 
 // SafeValue implements the redact.SafeValue interface.
 func (e Epoch) SafeValue() {}
+
+// Priority is used for replication admission control v2 (RAC2), where Raft
+// tracks admitted values per priority, and provides liveness by pinging when
+// the admitted values are lagging.
```

Comment:
This leaks RAC design decisions that are not relevant to the concept of "priority". Consider dropping this, and expanding on different priorities instead and what they mean. State that these are per-entry priorities, and what's the effect of one entry having a higher priority than another.